vars/sconsBuild.groovy

```diff
@@ -92,8 +93,12 @@ def call(Map config = [:]) {
         set_cwd = "cd ${config['target']}\n"
     }
 
+    def scons_exe = 'scons'
```

Comment:
Can we do some autodetection here, like check for the existence of a priority-descending list of alternatives.  Perhaps `scons-3`, followed by `scons` to start.

Reply:
Not until we have build supporting scons-3.   For now, we have to be explicit.  Otherwise, daos (which can't currently build with 3) will fail.

Reply:
Just so I'm clear, we're making this optional at this point because CaRT will build with `scons-3` but DAOS not yet?